src/hooks/useGallery/GalleryWithTheme/GalleryWithTheme.lazy.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+export const GalleryWithTheme = React.lazy(() =>
+    import('./GalleryWithTheme').then((module) => ({default: module.GalleryWithTheme})),
```

Comment:
Is the `default` member necessary here?

Reply:
Yes, because we don't have the default export in GalleryWithTheme